input/witnessgen.go

```diff
@@ -47,116 +47,119 @@ type StandardWitnessType uint16
 // WitnessType interface.
 var _ WitnessType = (StandardWitnessType)(0)
 
```

Comment:
nit: regarding the commit message, when using hyphens the following lines have a hanging indent => see contrib guidelines, though it your case you nit now bullet point list only one single point

Reply:
What do you mean by a hanging indent?

I just removed the bullet point and turned it into a normal paragraph instead.